modules/ROOT/pages/am-permissions.adoc

```diff
@@ -3,15 +3,20 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The *Access Management* section of Anypoint Platform enables you (as an Organization Administrator) to control user access to dashboards, reports, and metrics by providing two predefined product-specific permissions:
+Anypoint Monitoring has two product-specific permissions that control access to assets within Anypoint Monitoring:
 
-* Anypoint Monitoring Administrator. Allows users to view, create, modify, and delete all content in Anypoint Monitoring.
-+
-* Anypoint Monitoring Viewer. Allows users to view but not modify content in Anypoint Monitoring.
+Anypoint Monitoring Administrator:: Allows users to view, create, modify, and delete all content in Anypoint Monitoring.
+Anypoint Monitoring Viewer:: Allows users to view but not modify content in Anypoint Monitoring.
+
+The content you can access when you have the Anypoint Monitoring Administrator and Anypoint Monitoring Viewer permissions depends on other asset-specific permissions.
+To view or modify content related to specific apps and APIs, you also need the appropriate access permissions for those assets.
+For example, a user who has the Anypoint Monitoring Viewer permission who wants to view Anypoint Monitoring data for an on-premises app also needs the Read Applications permission in the same environment as the app.
```

Comment:
```suggestion
For example, a user who has the Anypoint Monitoring Viewer permission and who wants to view Anypoint Monitoring data for an on-premises app also needs the `Read Applications` permission in the same environment as the app.
```

Reply:
I put Read Applications in back ticks but don't know if that is consistent with the rest of the doc.

Reply:
We do not put permissions in backticks for now.